packages/design-system/package.json

```diff
@@ -28,7 +29,7 @@
     "@babel/preset-env": "^7.18.2",
     "@babel/preset-react": "^7.17.12",
     "@babel/preset-typescript": "^7.17.12",
-    "@blueprintjs/core": "^3.36.0",
+    "@blueprintjs/core": "3.33.0",
```

Comment:
why are we downgrading?

Reply:
not downgrading we are matching the same version as main repo, because it was throwing error.